docs/src/examples/addons/TextArea/Usage/index.js

```diff
@@ -9,7 +9,7 @@ const TextAreaUsageExamples = () => (
     <Segment>
       <Message warning>
         <Message.Header>Auto height</Message.Header>
-        We don't support `autoHeight` more. If you need this feature you can use{' '}
+        We don't support `autoHeight` anymore. If you need this feature you can use{' '}
```

Comment:
This is the change, just a minor verbiage adjustment in the docs for the TextArea page.